src/site/content/en/blog/betty-crocker/index.md

```diff
@@ -0,0 +1,340 @@
+that returns a `WakeLockSentinel` object.
+The browser can refuse the request for various reasons
+(for example, because the battery is too low),
+so it is a good practice to wrap the call in a `try…catch` statement.
+
+### Releasing a wake lock
+
+You also need a way to release a wake lock,
+which is achieved by calling the `release()` method of the `WakeLockSentinel` object.
+If you want to automatically release the wake lock after a certain period of time has passed,
+you can use `window.setTimeout()` to call `release()`, as shown in the example below.
+
+```js
+// The wake lock sentinel.
+let wakeLock = null;
+
+// Function that attempts to request a wake lock.
+const requestWakeLock = async () => {
+  try {
+    wakeLock = await navigator.wakeLock.request('screen');
+    wakeLock.addEventListener('release', () => {
+      console.log('Wake Lock was released');
+    });
+    console.log('Wake Lock is active');
+  } catch (err) {
+    console.error(`${err.name}, ${err.message}`);
+  }
+};
+
+// Request a wake lock…
+await requestWakeLock();
+// …and release it again after 5s.
+window.setTimeout(() => {
+  wakeLock.release();
+  wakeLock = null;
+}, 5000);
+```
+
+## The implementation
+
+With the new web app, users should be enabled to easily navigate through a recipe,
+complete steps, and even walk away without the screen locking.
+To achieve this goal, the team first built a quick front-end prototype
+as a proof the concept and to gather UX input.
+
+After the prototype proved useful, they designed a
+[Vue.js component](https://vuejs.org/v2/guide/components.html)
+that could be shared across all their brands ([BettyCrocker](https://www.bettycrocker.com/),
+[Pillsbury](https://www.pillsbury.com/), and [Tablespoon](https://www.tablespoon.com/)).
+Even though only Betty Crocker had a native app,
+the three sites do have a shared code base,
+so they were able to implement the component once, and deploy it everywhere,
+as shown in the screenshots below.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="www.bettycrocker.com.png" alt="BettyCrocker.com wake lock toggle" width="400">
+  <figcaption class="w-figcaption">BettyCrocker.com wake lock toggle.</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="www.pillsbury.com.png" alt="Pillsbury.com wake lock toggle" width="400">
+  <figcaption class="w-figcaption">Pillsbury.com wake lock toggle.</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="www.tablespoon.com.png" alt="Tablespoon.com wake lock toggle" width="400">
+  <figcaption class="w-figcaption">Tablespoon.com wake lock toggle.</figcaption>
+</figure>
+
+While focusing on the [`ViewModel` layer of the MVVM pattern](https://012.vuejs.org/guide/)
+in developing the component based on the new site's modernized framework,
+they developed with interoperability in mind
+to enable functionality on legacy and new frameworks of the site.
```

Comment:
This paragraph was hard for me to read. Can you rephrase it? I suggest breaking it up into smaller sentences.

Reply:
Broken down.